features/abi_apk_splits.feature

```diff
@@ -3,73 +3,89 @@ Feature: Plugin integrated in project with ABI APK splits
 @skip_agp4_1_or_higher
 Scenario: ABI Splits project builds successfully
     When I build "abi_splits" using the "standard" bugsnag config
-    Then I should receive 16 requests
+    And I wait to receive 16 requests
```

Comment:
Several features like this are heavily dependent on payload ordering.  I intend to address this in a separate PR targeting this branch before this branch is merged.